RxCocoa/iOS/UISearchBar+Rx.swift

```diff
@@ -47,6 +47,36 @@ extension UISearchBar {
         
         return ControlProperty(values: source, valueSink: bindingObserver)
     }
-}
-
+    
+    /**
+     Reactive wrapper for `searchBarSearchButtonClicked:` delegate event.
+     */
+    public var rx_searchTapped: ControlEvent<UISearchBar> {
```

Comment:
I'll rename it, and I get that you'd like the type to be consistent across the framework, but I honestly don't get why it couldn't be a `UISearchBar`, specially considering that that is what the original delegate returns.

Also, you wrote that it should be an `Observable`, why not `ControlProperty`?